### ISSUE TITLE
Documentation for `joinmarket.cfg`

### ISSUE THREAD

**Issue by dergigi** [joinmarket-webui/joinmarket-webui#30]
The full `joinmarket.cfg` config file contains many settings and multiple sections (daemon, blockchain, messaging, logging, timeout, policy, payjoin, yield generator), not all of which are immediately relevant to most users.

We should understand and document all settings, and break them down into two or three different sections, e.g. basic / advanced / expert.

Rough tasks list:

- [ ] Research `joinmarket.cfg`
- [ ] Rank-order by importance and/or use-case
- [ ] Write copy, find proper names and descriptions for settings (in cooperation with @GBKS)
- [ ] Update the [JM docs](https://github.com/JoinMarket-Org/joinmarket-clientserver/tree/master/docs) upstream

All of the above should serve as a base for implementing #12

**Comment by theborakompanioni** [comment]
Also, we need to come up with default values (or setting them by some other mechanism) for `max_cj_fee_abs` and/or `max_cj_fee_rel` as they are needed before sending payments via the jsonrpc api of `jmwalletd` (https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/d98cb36fb422edefab969a911564987f7878f021/jmclient/jmclient/wallet_rpc.py#L837-L845).

**Comment by dergigi** [comment]
Yes, sane default values would be great. To set them properly we need to do some order book parsing as sketched out in #14 I'm afraid. However, we can probably do some guesstimates first and hide some complexity from the user.

**Comment by dergigi** [comment]
Started some work on this, see the ["Configuration" page](https://github.com/joinmarket-webui/joinmarket-webui/wiki/Configuration:-UI-Settings-&-joinmarket.cfg) in the wiki.

**Comment by editwentyone** [comment]
this could also help: https://github.com/joinmarket-webui/joinmarket-webui/issues/72#issuecomment-1086865264

**Comment by parrth20** [comment]
hey i find this is issue still open can i work on this?

**Comment by theborakompanioni** [comment]
> hey i find this is issue still open can i work on this?

Yes! Of course! A list of all configs that makes sense to be included in the settings page would be awesome! Some are already noted in tickets, e.g. https://github.com/joinmarket-webui/jam/issues/742

But having a dedicated list and deciding on each config and subsequently creating tickets would be awesome 🧡
